src/Tests/PostfinancePaymentTest.php

```diff
@@ -76,13 +85,21 @@ protected function setUp() {
     // Create node with payment plugin configuration.
     $this->node = $this->drupalCreateNode(array(
       'type' => 'article',
-      $this->field_name => array(
+      $this->fieldName => array(
         'plugin_configuration' => array(
-          'currency_code' => 'XXX',
+          'currency_code' => 'CHF',
           'name' => 'payment_basic',
+          'payment_id' => NULL,
           'quantity' => '2',
           'amount' => '123',
-          'description' => 'pay me man',
+          'description' => 'Payment Description',
```

Comment:
Why are we adding this here, on the node? that's not configuration of the payment method, that's something that the user would fill in on the payment test form and is returned to us.